org.jacoco.build/pom.xml

```diff
@@ -708,37 +709,67 @@
       </build>
     </profile>
 
-    <!-- This profile is used to launch tests with compilation into specific bytecode version. -->
+    <!-- -->
```

Comment:
Is this empty comment on purpose?

Reply:
Yep - for visual separation of profile that follows this comment from one that is above.

Reply:
Some profiles are separated but some are not. I assume these are groups of profiles, right? Maybe add a short description into the comment for the profile group like with the profiles at the top?

Reply:
I added comments.